### ISSUE TITLE
HI i  will work on a very simple homework via your methods,more details  about my question is in comment   

### ISSUE THREAD

**Issue by resplendent-star** [t-taniai/LocalExpStereo#8]

**Comment by resplendent-star** [comment]
HI i  will work on a very simple homework via your methods，and my teachers tell my just use a very simple methods to  calculate the  matching cost ，for example  ，SSD（sum of square differences） methods， if  I  stored  the  matching cost  in    float v[disp][height][width],what should I do to  modified your code  ，as  in your code  you use  THE  cv::Mat  in thet function “convertVolumeL2R””fillOutOfView” I am a fresh man,so its too hard for me  to  make it,thanks you very much!!
Or  you could tell  once  I use the fread function to read v[disp][height][width] from my  own matching cost files,how could I translate the  float v[disp][height][width]   into  cv::Mat which used in you two functions?“convertVolumeL2R””fillOutOfView”
In other words,more simply ,as your said  in float v[disp][height][width] , a specific （disp，y，x）value  correspond  to we assume that in  such kind of  disp ，the matching cost between  （x，y）in left image  and （x-d，y）in right image （are my  comprehend is  the same as you？），that is the meaning of 3D float array v[disp][height][width] ，so what is the meaning of the cv：：Mat that you used in  “convertVolumeL2R””fillOutOfView”？What is the meaning of  the index of  Mat？Is the first index of Mat correspond to disp？2nd correspond to  row  ，3  rd correspond to col？
Thanks very much！！！！！
